src/arrayMethodSort.js

```diff
@@ -4,8 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a + '' < b + '') {
+        return -1;
+      }
+
+      return 1;
+    },
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length; j++) {
+        if (i !== j && compareFunction(this[i], this[j]) < 0) {
```

Comment:
The check `i !== j` is redundant as it's already ensured by the correct loop indices if the inner loop starts from `i + 1`.